.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -1,28 +1,27 @@
 name: Feature request
 description: Suggest an idea for this project
 title: '[Feature request]: '
-labels:
-assignees: ''
+labels: [feature]
```

Let's start with the `enhancement` label, already used in this repo: (https://github.com/devopness/devopness/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) 

```suggestion
labels: [enhancement]
```

---

Label altered.